frontend/app/components/account-form.js

```diff
@@ -85,6 +115,18 @@ export default class AccountForm extends BaseFormComponent {
     /* eslint-disable no-undef  */
     $(this.modalElement).modal("hide");
     /* eslint-enable no-undef  */
-    window.location.replace("/accounts/" + savedRecords[0].id);
+
+    if (this.isEditView) {
+      let href = window.location.href
+      window.location.replace(href.substring(0, href.search('#')));
+    } else {
+      window.location.replace("/accounts/" + savedRecords[0].id);
+    }
```

Comment:
Not, if you edit an account from` /groups/id`. There is a Edit-Button aswell. So when editing there, it will return to `/groups/id` and not `/accounts/id`